src/converter/one/util/ResourceProcessor.java

```diff
@@ -0,0 +1,33 @@
+package one.util;
+
+import java.io.*;
+
+public class ResourceProcessor {
+
+    public static String getResource(String name) {
```

Comment:
FlameGraph also has a similar method. Can it be de-duplicated?

Reply:
fixed